include/UI/UIView.h

```diff
@@ -181,13 +208,15 @@ class UIView
   glm::vec3 m_clip_ypr;
   glm::vec3 m_clip_pos;
 
-  int m_table_selection;
+  int m_trnfnc_table_selection;
+
+  int m_camera_poi_table_selection;
 
   bool m_initialized;
 
-  bool m_trn_fct_opitions_window;
+  bool m_trn_fct_options_window;
 
-  bool m_save_trnfct_open;  
+  bool m_save_trnfct_open;
```

Comment:
I'd be consistent in the use of either `trn_fct` `transfer_function` `trnfnc` or `trnfct`, but not mix and match

Reply:
I renamed it to transfer_function